### ISSUE TITLE
MapReader: Create a function which allows us to load and safe huggingface models directly into MapReader

### ISSUE THREAD

**Issue by rwood-97** [alan-turing-institute/hut101#19]
### Original GitHub issue

https://github.com/maps-as-data/MapReader/issues/192

### Mentor

@rwood-97

### Project overview

MapReader is a open-source python library for exploring and analyzing map images at scale.

It contains two different pipelines:

- Classification pipeline: This pipeline enables users to fine-tune a classification model and predict the labels of patches created from a parent image.
- Text spotting pipeline: This pipeline enables users to detect and recognize text in map images.

MapReader was developed in the [Living with Machines](https://livingwithmachines.ac.uk/) project to analyze large collections of historical maps but is a generalizable computer vision tool which can be applied to any images in a wide variety of domains.

### Skills / technologies used

Python, HuggingFace, PyTorch, 

### Difficulty level

Easy

**Comment by jhwisdom** [comment]
Hi, I would like to join this task. - Jihye

**Comment by jhwisdom** [comment]
Hi @rwood-97 , I made a pull request for this task. Thank you for reviewing! 
[Pull request](https://github.com/maps-as-data/MapReader/pull/577)

**Comment by rwood-97** [comment]
Done! Thanks @jhwisdom
